extensions/spring-xd-extension-kafka/src/main/java/org/springframework/integration/x/kafka/KafkaPartitionAllocator.java

```diff
@@ -132,6 +132,7 @@ public void afterPropertiesSet() throws Exception {
 		if (STARTED.equals(this.client.getState())) {
 			EnsurePath ensurePath = new EnsurePath(partitionDataPath);
 			ensurePath.ensure(client.getZookeeperClient());
+			// ignore sonar's complaint about this not being synchronized
```

Comment:
Are you saying that this sole comment stops Sonar from complaining? Does it have to follow a given pattern?